app/pages/print/print.js

```diff
@@ -0,0 +1,380 @@
+/**
```

Comment:
ahh crap - yeah thats a relic of the original POC work that I need to cleanup. Ta